srv/modules/runners/push.py

```diff
@@ -137,9 +137,14 @@ def _custom(self, custom):
             if not os.path.isfile(custom):
                 log.info("Writing {}".format(custom))
                 with open(custom, "w") as yml:
-                    yml.write("# Customizations for {}\n".format(custom))
-    
-    
+                    custom_split = custom.split("stack")
+                    custom_for = "{}{}{}".format(
+                            custom_split[0],
+                            "stack/default",
+                            custom_split[1])
+                    yml.write("# Customizations for {}\n".format(custom_for))
```

Comment:
We could probably do it in one shot as something like 
```py
yml.write("customizations for {0}{1}{2}\n".format(split[0]...)
```
Though that doesn't describe it any better :)